src/main/java/callofcactus/Administration.java

```diff
@@ -148,11 +148,19 @@ public void setGodmode(boolean godmode) {
     }
 
     public HumanCharacter getLocalPlayer() {
+        for (MovingEntity movingEntity : this.getMovingEntities()){
+            if (movingEntity instanceof HumanCharacter){
+                if (movingEntity.getID() == localPlayer.getID()){
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [These nested if statements could be combined](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1572736274/issues/source?bid=2979575&fileBranchId=3022095#l153)